infra/build/functions/build_lib.py

```diff
@@ -186,6 +186,38 @@ def download_corpora_steps(project_name, testing):
   return steps
 
 
+def download_coverage_data_steps(project_name, latest, bucket_name, testing):
```

Comment:
I'm going to try to think of a way to share code with download_corpora_steps since it looks like this code was copied.